cmd/clusterctl/client/config/providers_client.go

```diff
@@ -44,6 +44,7 @@ const (
 	DockerProviderName         = "docker"
 	DOProviderName             = "digitalocean"
 	GCPProviderName            = "gcp"
+	HarvesterProviderName      = "harvester-harvester"
```

Comment:
Well, all the orgs `harvester` , `rancher` and `rancher-sandbox` belong to SUSE. I think it is more sensible to use the `harvester` org for harvester-related projects. In the future, after this project is streamlined, it will probably be moved to the `harvester` org in GitHub.

Reply:
Please let's add a comment tracking this, e.g.
```suggestion
        // Note: harvester is currently on rancher-sandbox, but the plan is to move it on the harvester GitHub org (also owned by Rancher)
	HarvesterProviderName      = "harvester-harvester"
```